app/components/home/stacking-participation-card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import { Box, Flex, Text, Button } from '@blockstack/ui';
+
+import routes from '@constants/routes.json';
+import btcPodium from '../../assets/images/btc-podium.svg';
```

Comment:
Does it matter if we import raw svg vs having a React component for this, a la `@blockstack/ui/icons` and many other icons in this repo?

Reply:
I imported this as an image, it's used in like a promotional sense, rather than a meaningful icon sense.